### ISSUE TITLE
Unable to access context menus.

### ISSUE THREAD

**Issue by ladza** [pyatom/pyatom#74]
Steps-to-reproduce:
1. Invoke a context menu in any application by performing a right mouse button click or ctrl + left mouse button click.
2. Try to read the contents of context menu using atomac.

Attaching script with minimal test case.
https://gist.github.com/ladza/9329103#file-gistfile1-py

**Comment by nagappan** [comment]
How its shown in Accessibility inspector ? I guess it should be part of main window.

**Comment by ladza** [comment]
Indeed, it is shown exactly as you say. However atomac does not list any menus in window. There must be something in the way atomac populates the application map, that causes it fails to report existance of AXMenu refering to context menu.

**Comment by dwabyick** [comment]
Has anyone figured this out?  We're still struggling to find the context menu.
